CleanCode/java-introduction-master/src/main/java/pl/futurecollars/lesson5/task5/Bicycle.java

```diff
@@ -0,0 +1,8 @@
+package pl.futurecollars.lesson5.task5;
+
+public class Bicycle extends Vehicle {
```

Comment:
To taki rower bez silnika - czy wtedy faktycznie chcemy dziedziczyć po czymś co ma silnik?

Reply:
2 opcje:
1. Vehicle jednak nie ma silnika (może jakiś interfejs?)
2. Vehicle ma silnik, ale wtedy nie dziedziczymy.